### PR TITLE
Remove company merging tool feature flag

### PR DESCRIPTION
**Pull request by reupen**
### Description of change

As the merging tool is now functional, this removes the feature flag. There are still some front-end changes to make, but the feature flag is not required for that.

### Checklist

* [x] Has a new newsfragment been created? Check [changelog/README.rst](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.rst) for instructions
* [ ] Have any relevant search models been updated?
* [ ] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

**Comment by reupen** [comment]
I have put this back to work in progress, as there are a few other small tweaks that we want to make.

**Comment by reupen** [comment]
Updated with a few more small changes.

File: changelog/company/merging-tool.feature [hunk 1]
```diff
@@ -0,0 +1,4 @@
+A tool for merging duplicate companies was added to the admin site. This tool moves contacts and interactions from one
+company to another, and archives the the company that the contacts and interactions where moved from. The tool is
```
Comment (marcuspp) [inline changelog/company/merging-tool.feature]: typo `the the`

**Comment by marcuspp** [comment]
Other than the typo all seems good.

File: datahub/core/templatetags/datahub_extras.py [hunk 2]
```diff
@@ -36,12 +37,22 @@ def admin_change_url(obj):
...
     Template filter to generate the URL to the admin change page for a model object.
 
-    Usage example:
+    Usage examples:
 
+      Normal anchor tag:
       {{ target_company|admin_change_link }}
+
+      With target="_blank":
+      {{ target_company|admin_change_link:True }}
```
Comment (elcct) [inline datahub/core/templatetags/datahub_extras.py]: Is there a way to be explicit about target_blank in the attribute?

Comment (marcofucci) [reply datahub/core/templatetags/datahub_extras.py]: yeah the `True` is not really great, I don't think there's a way with filters, you would have to use simple tags and assign to a variable which could be better?

Comment (reupen) [reply datahub/core/templatetags/datahub_extras.py]: I've changed it to be a (simple) tag.

**Pull request merged**